how-to-guides/KyloDatasources.rst

```diff
@@ -0,0 +1,80 @@
+========================
+Kylo Datasources Guide
+========================
+
+Introduction
+====================
+
+Kylo can manage the creation and usage of Nifi RDBMS data source configurations, through a simple `Data Source UI <http://localhost:8400/index.html#!/datasources>`_.
+
+JDBC
+=============
+
+.. note:: Permissions for the jars are separate per the type of process running.
+
+Locations
+-----------
+
+    +----------------------------------+------------------------------------------------------------------------------------------------------------------+
+    | $NIFI_HOME/nifi-<version>/lib    | Needed by Nifi for the DBCPConnectionPool. Can set a custom path, but need to set `Database Driver Location(s)`  |
```

Comment:
It would be good to have a permanent location that won't get lost in a NiFi upgrade. Another possible location is $NIFI_HOME/data/lib/.